src/app/[entity]/[repo]/issues/new/page.tsx

```diff
@@ -1,7 +1,284 @@
+"use client";
+
+import {
+  validateEvent,
+  verifySignature,
+  signEvent,
+  getEventHash,
+  getPublicKey
+} from 'nostr-tools'
+
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage
+} from "@/components/ui/avatar";
+import { Button } from "@/components/ui/button";
+import useSession from "@/lib/nostr/useSession";
+
+import {
+  useCallback,
+  useRef,
+  useState
+} from "react";
+
+import { Input } from "@/components/ui/input";
+
+import { useRouter } from "next/navigation";
+import { Check, ChevronDown, Edit, Settings } from 'lucide-react';
+import { TextArea } from '@/components/ui/textarea';
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuGroup,
+  DropdownMenuItem
+} from '@/components/ui/dropdown-menu';
+import Link from 'next/link';
+
 export default function RepoIssueNewPage() {
+  const router = useRouter();
+  const titleRef = useRef<HTMLInputElement>(null);
+  const commentRef = useRef<HTMLTextAreaElement>(null);
+  const assigneesFilterRef = useRef<HTMLInputElement>(null);
+  const labelsFilterRef = useRef<HTMLInputElement>(null);
+  const [errorMsg, setErrorMsg] = useState<string>('');
+  const [selectedLabels, setSelectedLabels] = useState<string[]>([]);
+  const { picture, initials, isLoggedIn } = useSession();
+
+  const handleSubmit = useCallback(
+    (e: React.FormEvent<HTMLFormElement>) => {
+      e.preventDefault();
+
+      // todo: check the correct localstorage key for nsec
+      let privateKey = localStorage.getItem('nostr:privkey')
+
+      if (!isLoggedIn || !privateKey) {
+        setErrorMsg('Sign in with your private key to create issues.')
+        setTimeout(() => {
+          setErrorMsg('')
+        }, 6000)
+        return
+      }
+
+      let event = {
+        kind: 1,
+        created_at: Math.floor(Date.now() / 1000),
+        tags: [],
+        content: `${titleRef.current?.value}\n\n${commentRef.current?.value}` || "",

```

Comment:
```suggestion
        content: `${titleRef.current?.value || ""}\n\n${commentRef.current?.value || ""}` || "",
```
will remove the eslint error `Invalid type "string | undefined" of template literal expression.`